ui/pages/onboarding-flow/import-srp/index.scss

```diff
@@ -1,5 +1,21 @@
 .import-srp {
-  width: 625px;
+  width: 840px;
+
+  @media screen and (max-width: 349px) {
```

Comment:
What is the origin of these screen sizes? I thought we were using `767px`, `1270px` as our media query breakpoints, with `357px` being our minimum supported screen size.